libs/commons/Logs_.ml

```diff
@@ -50,6 +50,63 @@ let default_skip_libs =
 (* used for testing *)
 let in_mock_context = ref false
 
+(*****************************************************************************)
+(* String tags *)
+(*****************************************************************************)
+(*
+   The interface of the Logs.Tag module is complicated. Here we assume
```

Comment:
This daniel buenzli is making good libraries (Logs, Cmdliner, Fpath), but they are not super easy to use at first ... the learning curve is deep.